src/test/java/uk/gov/companieshouse/confirmationstatementapi/configuration/InterceptorConfigRouteMatchingTest.java

```diff
@@ -51,17 +59,11 @@ void interceptorsMatchIntendedRoutesTest() throws Exception {
         testCases.put("/transactions/12345/confirmation-statement/12345/shareholders", CS_ID_INTERCEPTORS);
         testCases.put("/transactions/12345/confirmation-statement/12345/register-locations", CS_ID_INTERCEPTORS);
         // internal endpoints
-        testCases.put("/private/transactions/12345/confirmation-statement/12345/filings",
-            Set.of("LoggingInterceptor", "InternalUserInterceptor", "TransactionInterceptor", "FilingInterceptor"));
-        testCases.put("/transactions/12345/confirmation-statement/12345/costs",
-            Set.of("LoggingInterceptor", "InternalUserInterceptor", "TransactionInterceptor", "TransactionIdValidationInterceptor",
-                "SubmissionInterceptor", "SubmissionIdValidationInterceptor"));
-        testCases.put("/private/confirmation-statement/company/12345/registered-email-address",
-            Set.of("LoggingInterceptor", "InternalUserInterceptor", "CompanyNumberValidationInterceptor"));
-
-
-        for (String requestPath : testCases.keySet()){
+        testCases.put("/private/transactions/12345/confirmation-statement/12345/filings", Set.of("LoggingInterceptor", "InternalUserInterceptor", "TransactionInterceptor", "FilingInterceptor"));
+        testCases.put("/transactions/12345/confirmation-statement/12345/costs", Set.of("LoggingInterceptor", "InternalUserInterceptor", "TransactionInterceptor", "TransactionIdValidationInterceptor", "SubmissionInterceptor", "SubmissionIdValidationInterceptor"));
+        testCases.put("/private/confirmation-statement/company/12345/registered-email-address", Set.of("LoggingInterceptor", "InternalUserInterceptor", "CompanyNumberValidationInterceptor"));
 
+        for (String requestPath : testCases.keySet()) {
```

Comment:
Not a biggy so no problem being left in but maybe convert to using something like
testCases.keySet().stream().forEach(requestPath -> {.....